docs/registry.md

```diff
@@ -0,0 +1,58 @@
+# Configure Image Registry for CRI-containerd
+This document describes the methods how to configure the image registry for CRI-containerd.
+
+In CRI-containerd, `docker.io` is the default image registry. You can also set  up other image registries similar to docker. CRI-containerd provides two ways to set up the image registry.
```

Comment:
Add TODO to remove the document for standalone mode and also add `(containerd/containerd#2120)`.